SP-4011 (#9)
stackpulse/playbooks: local-diagnostics/README.md

> # Local Diagnostics

Author: I'm no sure about this name
let me know what you think

<img src="../../images/ssh.png" width="100" alt="diagnostics">

![uses-slack](https://img.shields.io/static/v1?label=uses&message=Slack&style=flat&logo=slack&color=4A154B)

[![Import to StackPulse](../../images/open_in_stackpulse.svg)](https://app.stackpulse.io/playbook/create#https://github.com/stackpulse/playbooks/blob/master/local-diagnostics/playbook.yaml)

> This playbook queries given host memory, storage and cpu usage using well know linux commands  - df, ps and vmstat.

Reviewer: ```suggestion
This playbook queries utilization of CPU, memory and storage for a given host and sends the output to Slack recipients of choice.
```

> It then sends that output to Slack recipients of your choice as a snippet.

Reviewer: ```suggestion
```

## Your benefits

> - Automatically retrieve local information regarding your machine with common linux commands - df, ps and vmstat.

Reviewer: ```suggestion
- Triage production hosts without having to provide multiple people direct access to production.
```
